workstation-bootstrap/requirements.in

```diff
@@ -18,3 +18,4 @@ flit_core >=2,<3
 
 # For maintaining requirements files
 pip-tools>=6
+click==8.0.3
```

Comment:
Since this is an implicit dependency, we shouldn't keep it in requirements.in. Now that it's at 8.0.3, you can remove it from here and it'll stay pinned at 8.0.3.

Reply:
:bulb: That wasn't in my radar. Thanks for explaining @legoktm !